historical/tests/test_commons.py

```diff
@@ -110,7 +110,7 @@ def test_deserialize_current_record_to_current_model(historical_role, current_s3
             'arn': bucket['arn']
         }),
         eventName='INSERT'), default=serialize))
-    ddb_record['sns_too_big'] = True
+    ddb_record['event_too_big'] = True
```

Comment:
Could import the `CONSTANT` variable here.

Reply:
Will do.